docs/reference/tab-widgets/inference-api/infer-api-ingest-pipeline-widget.asciidoc

```diff
@@ -77,12 +83,12 @@ include::infer-api-ingest-pipeline.asciidoc[tag=azure-openai]
   </div>
   <div tabindex="0"
        role="tabpanel"
-       id="infer-api-ingest-azure-ai-studio-tab"
-       aria-labelledby="infer-api-ingest-azure-ai-studio"
+       id="infer-api-ingest-mistral-tab"
```

Comment:
The widget for `azure-ai-studio` has been edited out. This is the reason why the tutorial page is not rendered correctly.

Reply:
No problem at all, thank you for writing this content up!